person.rb

```diff
@@ -27,6 +30,15 @@ def can_use_services
     end
   end
 
+  def correct_name
+    @name
+  end
+
+  def add_rental(date, book)
+    rental = Rental.new(date, book, self)
+    @rentals << rental
```

Comment:
- Hey, you should only instantiate the class for now. Kindly remove this line as you'd do that later in the next milestone.